### ISSUE TITLE
2019-11-28 tc008 M1SS, tc010 M1SD, tc011 M1SS tc012 M1SS, tc014 M2SS missing partial AUCs from AUCT1_T2 and incremental AUCs from AUCT. Interpolated and Extrapolated values all missing from Estimated concentration dataset

### ISSUE THREAD

**Issue by tensfeldt** [tensfeldt/openNCA#65]
With tc010 M1SD, tc011 M1SS, tc012 M1SS, tc014 M2SS and tc008 M1SS no partial AUCs nor incremental AUCs are being recorded in the output parameter dataset.
Further there are NO interpolated or extrapolated values in the estimated concentration dataset. 

** tc008 parameter results are missing partial AUCs requested and incremental AUCs

mct requests the following partial AUCs:

```
 AUC.1.T1 AUC.1.T2 AUC.2.T1 AUC.2.T2 AUC.3.T1 AUC.3.T2 AUC.4.T1 AUC.4.T2 AUC.5.T1 AUC.5.T2
1        0       12        0        8       13       24        0       10       12       30
```

*** mct specified AUCT and AUCT1_T2 in PARAMETERLIST:

```
> mct$PARAMETERLIST
[1] "AUCALL;AUCALLDN;AUCINFO;AUCINFOC;AUCINFODN;AUCINFP;AUCINFPC;AUCINFPDN;AUCLAST;AUCLASTi;AUCLASTDNi;AUCLASTCi;AUCT;AUCT1_T2;AUCTAUi;AUCTAUDNi;AUCXPCTO;AUCXPCTP;AUMCTAUi;CAVi;CLASTi;CLFO;CLFTAUi;CLFTAUWi;CLO;CLRTAUi;CMAXi;CMAXCi;CMAXDNi;CMINi;DOSECi;CTROUGHi;CTROUGHENDi;DIi;DOSEi;KEL;KELNOPT;KELRSQ;KELRSQA;KELTMHI;KELTMLO;LASTTIME;MRTEVIFOi;MRTEVIFPi;PTFi;PTRi;PTROUGHRi;PTROUGHREND;TAU;TAUi;THALF;THALFF;TLAG;TLAST;TMAXi;TMINi;TOLD;VZFP;VZFTAUi;VZFTAUWi"
```

** tc010 M1 SD

*** partial areas requested for tc010

```
> mct[, names(j)]
  AUC.1.T1 AUC.1.T2 AUC.2.T1 AUC.2.T2 AUC.3.T1 AUC.3.T2 AUC.4.T1 AUC.4.T2 AUC.5.T1 AUC.5.T2 AUC.6.T1 AUC.6.T2
1        0       12        8       48        0       64       12      130       24       96       60      124

```

*** tc010 estimated concentration dataset is missing any of the interpolated and extrapolated concentration values

```

> e
    PKDATAROWID     SDEID TIME     CEST_KEL CEST_INT CEST_EXT CEST_C0 CEST_TLAST
1      22289754   2925168   24   1.70263761       NA       NA      NA         NA
2      22289755   2925168   48   0.58780724       NA       NA      NA         NA
3      22289756   2925168   72   0.20293065       NA       NA      NA 0.20293065
4      22289571  40876220   16   0.49544432       NA       NA      NA         NA
5      22289572  40876220   24   0.36517471       NA       NA      NA         NA
6      22289573  40876220   48   0.14622386       NA       NA      NA         NA
7      22289574  40876220   72   0.05855120       NA       NA      NA 0.05855120
8      22289531  63536292   12   0.51896071       NA       NA      NA         NA
9      22289532  63536292   16   0.36933771       NA       NA      NA         NA
10     22289533  63536292   24   0.18706907       NA       NA      NA 0.18706907
11     22289597 111299859   16   0.58377725       NA       NA      NA         NA
12     22289598 111299859   24   0.37399383       NA       NA      NA         NA
13     22289599 111299859   48   0.09833684       NA       NA      NA 0.09833684
14     22289765 112146587   12   1.59483827       NA       NA      NA         NA
15     22289766 112146587   16   1.14287770       NA       NA      NA         NA
16     22289767 112146587   24   0.58690253       NA       NA      NA         NA
17     22289768 112146587   48   0.07948124       NA       NA      NA 0.07948124
18     22290452 123811839   12  44.59385840       NA       NA      NA         NA
19     22290453 123811839   16  36.10132904       NA       NA      NA         NA
20     22290454 123811839   24  23.66025768       NA       NA      NA         NA
21     22290455 123811839   48   6.66052265       NA       NA      NA         NA
22     22290456 123811839   72   1.87498220       NA       NA      NA         NA
23     22290457 123811839   96   0.52782018       NA       NA      NA         NA
24     22290458 123811839  120   0.14858495       NA       NA      NA 0.14858495
25     22291157 145593586   48  18.32531818       NA       NA      NA         NA
26     22291158 145593586   72   5.87505987       NA       NA      NA         NA
27     22291159 145593586   96   1.88353229       NA       NA      NA         NA
28     22291160 145593586  120   0.60385664       NA       NA      NA 0.60385664
29     22290934 202367199   16 155.89036680       NA       NA      NA         NA
30     22290935 202367199   24 107.16512307       NA       NA      NA         NA
31     22290936 202367199   48  34.81416886       NA       NA      NA         NA
32     22290937 202367199   72  11.30989560       NA       NA      NA         NA
33     22290938 202367199   96   3.67418619       NA       NA      NA         NA
34     22290939 202367199  120   1.19361351       NA       NA      NA 1.19361351
35     22290518 233891605   16  41.50390320       NA       NA      NA         NA
36     22290519 233891605   24  27.33064016       NA       NA      NA         NA
37     22290520 233891605   48   7.80427191       NA       NA      NA         NA
38     22290521 233891605   72   2.22851202       NA       NA      NA         NA
39     22290522 233891605   96   0.63635223       NA       NA      NA         NA
40     22290523 233891605  120   0.18171056       NA       NA      NA 0.18171056
41     22289791 257914734   12   1.66768192       NA       NA      NA         NA
42     22289792 257914734   16   1.47132691       NA       NA      NA         NA
43     22289793 257914734   24   1.14525209       NA       NA      NA         NA
44     22289794 257914734   48   0.54010330       NA       NA      NA         NA
45     22289795 257914734   72   0.25471386       NA       NA      NA 0.25471386
46     22290260 355441192   48   2.64457864       NA       NA      NA         NA
47     22290261 355441192   72   0.96307502       NA       NA      NA         NA
48     22290262 355441192   96   0.35072260       NA       NA      NA         NA
49     22290263 355441192  120   0.12772249       NA       NA      NA 0.12772249
50     22290686 409149207   12  92.37008404       NA       NA      NA         NA
51     22290687 409149207   16  75.56168235       NA       NA      NA         NA
52     22290688 409149207   24  50.56408802       NA       NA      NA         NA
53     22290689 409149207   48  15.15178707       NA       NA      NA         NA
54     22290690 409149207   72   4.54031034       NA       NA      NA         NA
55     22290691 409149207   96   1.36052717       NA       NA      NA         NA
56     22290692 409149207  120   0.40768891       NA       NA      NA 0.40768891
57     22290960 427589065   16 140.81814484       NA       NA      NA         NA
58     22290961 427589065   24  96.88854127       NA       NA      NA         NA
59     22290962 427589065   48  31.55831549       NA       NA      NA         NA
60     22290963 427589065   72  10.27910281       NA       NA      NA         NA
61     22290964 427589065   96   3.34808601       NA       NA      NA         NA
62     22290965 427589065  120   1.09053097       NA       NA      NA 1.09053097
63     22289816 443820693    8   2.66007299       NA       NA      NA         NA
64     22289817 443820693   12   2.22883711       NA       NA      NA         NA
65     22289818 443820693   16   1.86751073       NA       NA      NA         NA
66     22289819 443820693   24   1.31109047       NA       NA      NA         NA
67     22289820 443820693   48   0.45367131       NA       NA      NA         NA
68     22289821 443820693   72   0.15698204       NA       NA      NA 0.15698204
69     22290077 463304641   12   5.87824590       NA       NA      NA         NA
70     22290078 463304641   16   5.06153051       NA       NA      NA         NA
71     22290079 463304641   24   3.75275385       NA       NA      NA         NA
72     22290080 463304641   48   1.52951851       NA       NA      NA         NA
73     22290081 463304641   72   0.62338938       NA       NA      NA 0.62338938
74     22289975 471081571   24   2.72023498       NA       NA      NA         NA
75     22289976 471081571   48   0.89584521       NA       NA      NA         NA
76     22289977 471081571   72   0.29502548       NA       NA      NA 0.29502548
77     22290778 499820574   16 119.86811280       NA       NA      NA         NA
78     22290779 499820574   24  85.89601519       NA       NA      NA         NA
79     22290780 499820574   48  31.60683899       NA       NA      NA         NA
80     22290781 499820574   72  11.63025163       NA       NA      NA         NA
81     22290782 499820574   96   4.27954067       NA       NA      NA         NA
82     22290783 499820574  120   1.57472675       NA       NA      NA 1.57472675
83     22289780 500091016   24   1.81167940       NA       NA      NA         NA
84     22289781 500091016   48   0.50344554       NA       NA      NA         NA
85     22289782 500091016   72   0.13990191       NA       NA      NA 0.13990191
86     22290248 500805530   72   1.40793645       NA       NA      NA         NA
87     22290249 500805530   96   0.61846363       NA       NA      NA         NA
88     22290250 500805530  120   0.27167224       NA       NA      NA 0.27167224
89     22290767 503192693   48  16.58665461       NA       NA      NA         NA
90     22290768 503192693   72   7.10451147       NA       NA      NA         NA
91     22290769 503192693   96   3.04305385       NA       NA      NA         NA
92     22290770 503192693  120   1.30342203       NA       NA      NA 1.30342203
93     22290012 507682703   12   6.51779455       NA       NA      NA         NA
94     22290013 507682703   16   5.05911956       NA       NA      NA         NA
95     22290014 507682703   24   3.04805989       NA       NA      NA         NA
96     22290015 507682703   48   0.66660590       NA       NA      NA         NA
97     22290016 507682703   72   0.14578566       NA       NA      NA 0.14578566
98     22290741 518707064   48  22.18560821       NA       NA      NA         NA
99     22290742 518707064   72   9.73309608       NA       NA      NA         NA
100    22290743 518707064   96   4.27002759       NA       NA      NA         NA
101    22290744 518707064  120   1.87331303       NA       NA      NA 1.87331303
102    22290482 524844737   72   3.59822527       NA       NA      NA         NA
103    22290483 524844737   96   1.77890002       NA       NA      NA         NA
104    22290484 524844737  120   0.87945724       NA       NA      NA 0.87945724
105    22290206 539272807   16  10.17403906       NA       NA      NA         NA
106    22290207 539272807   24   7.30945717       NA       NA      NA         NA
107    22290208 539272807   48   2.71056796       NA       NA      NA         NA
108    22290209 539272807   72   1.00516064       NA       NA      NA         NA
109    22290210 539272807   96   0.37274399       NA       NA      NA         NA
110    22290211 539272807  120   0.13822475       NA       NA      NA 0.13822475
111    22291001 565486844   48  50.14359219       NA       NA      NA         NA
112    22291002 565486844   72  23.26605742       NA       NA      NA         NA
113    22291003 565486844   96  10.79518646       NA       NA      NA         NA
114    22291004 565486844  120   5.00884394       NA       NA      NA 5.00884394
115    22290440 577177821   16  35.48137175       NA       NA      NA         NA
116    22290441 577177821   24  24.59704807       NA       NA      NA         NA
117    22290442 577177821   48   8.19465651       NA       NA      NA         NA
118    22290443 577177821   72   2.73009977       NA       NA      NA         NA
119    22290444 577177821   96   0.90954938       NA       NA      NA         NA
120    22290445 577177821  120   0.30302192       NA       NA      NA 0.30302192
121    22290911 579271281   72   4.68087054       NA       NA      NA         NA
122    22290912 579271281   96   1.99809352       NA       NA      NA         NA
123    22290913 579271281  120   0.85291350       NA       NA      NA 0.85291350
124    22290040 582963447   24   4.09698709       NA       NA      NA         NA
125    22290041 582963447   48   1.15916411       NA       NA      NA         NA
 [ reached 'max' / getOption("max.print") -- omitted 82 rows ]
```

*** tc010 parameters 

```
> r
      SDEID DOSE DOSEC    CMAX FLGACCEPTPREDOSE CMIN  CLAST       CEST   CMAXDN TMAX TMIN TLAST TLAG        KEL KELTMLO KELTMHI KELNOPT
1   2925168    3 3e+06    3.60                1    0 0.2050 0.20293065 1.200000    8    0    72    0 0.04431395      24      72       3
2  40876220    1 1e+06    1.30                1    0 0.0677 0.05855120 1.300000    8    0    72    0 0.03813488      16      72       4
3  63536292    1 1e+06    1.08                1    0 0.1910 0.18706907 1.080000    8    0    24    0 0.08502919      12      24       3
4 111299859    1 1e+06    1.46                1    0 0.0975 0.09833684 1.460000    8    0    48    0 0.05566002      16      48       3
5 112146587    3 3e+06    3.19                1    0 0.0776 0.07948124 1.063333    4    0    48    0 0.08330574      12      48       4
6 123811839  100 1e+08  399.00                1    0 0.2810 0.14858495 3.990000    1    0   120    0 0.05281662      12     120       7
7 145593586  100 1e+08  150.00                1    0 0.6530 0.60385664 1.500000    2    0   120    0 0.04739864      48     120       4
8 202367199  300 3e+08 1340.00                1    0 1.6600 1.19361351 4.466667    2    0   120    0 0.04684777      16     120       6
     KELRSQ   KELRSQA FLGACCEPTKEL     THALF THALFF LASTTIME FLGACCEPTTAU      AUCALL AUCALLDN     AUCLAST    AUCLASTC AUCLASTDN
1 0.9997271 0.9994541            1 15.641738      0       72            0    93.56369 31.18790    93.56369    93.56369  31.18790
2 0.9655010 0.9482515            1 18.176198      0       72            0    23.07816 23.07816    23.07816    23.07816  23.07816
3 0.9889092 0.9778184            1  8.151873      1       72            0    15.17517 15.17517    12.88317    12.88317  12.88317
4 0.9988961 0.9977922            1 12.453232      0       72            0    23.82186 23.82186    22.65186    22.65186  22.65186
5 0.9951571 0.9927356            1  8.320521      0       72            0    47.69731 15.89910    46.76611    46.76611  15.58870
6 0.9636440 0.9563728            1 13.123657      0      120            0  3185.37920 31.85379  3185.37920  3185.37920  31.85379
7 0.9966203 0.9949304            1 14.623778      0      120            0  2692.11817 26.92118  2692.11817  2692.11817  26.92118
8 0.9835298 0.9794122            1 14.795736      0      120            0 13167.20305 43.89068 13167.20305 13167.20305  43.89068
     AUMCLAST     AUCINFO     AUCINFP    AUCINFOC    AUCINFPC AUCINFODN AUCINFPDN    AUMCINFO    AUMCINFP  MRTLAST MRTEVIFO MRTEVIFP
1   2018.9186    98.18977    98.14307    98.18977    98.14307  32.72992  32.71436   2456.3899   2451.9739 21.57801 25.01676 24.98367
2    481.4440    24.85344    24.61354    24.85344    24.61354  24.85344  24.61354    655.8166    632.2523 20.86145 26.38736 25.68718
3    124.8745    15.12946    15.08323    15.12946    15.08323  15.12946  15.08323    205.2033    203.5500  9.69284 13.56316 13.49512
4    360.5995    24.40357    24.41860    24.40357    24.41860  24.40357  24.41860    476.1530    477.1448 15.91920 19.51161 19.54021
5    588.6315    47.69762    47.72020    47.69762    47.72020  15.89921  15.90673    644.5257    645.8808 12.58671 13.51274 13.53474
6  33597.3845  3190.69949  3188.19242  3190.69949  3188.19242  31.90699  31.88192  34336.5515  33988.2354 10.54737 10.76145 10.66066
7  63972.8984  2705.89494  2704.85813  2705.89494  2704.85813  27.05895  27.04858  65916.7678  65770.4764 23.76304 24.36043 24.31568
8 174414.5254 13202.63697 13192.68161 13202.63697 13192.68161  44.00879  43.97561 179422.9588 178015.8108 13.24613 13.58993 13.49353
    AUCXPCTO    AUCXPCTP  AUMCXPTO  AUMCXPTP     CLFO    CLFOW     CLFP    CLFPW      VZFO     VZFOW      VZFP     VZFPW CONC1   CONC2
1  4.7113693  4.66602992 17.809524 17.661498 30.55308 328.5277 30.56762 328.6841  689.4687  7.413642  689.7967  7.417169     0   1.380
2  7.1429844  6.23791345 26.588617 23.852550 40.23588 406.4230 40.62805 410.3844 1055.0937 10.657512 1065.3776 10.761390     0   0.176
3 14.8471079 14.58611253 39.145933 38.651677 66.09621 869.6870 66.29880 872.3526  777.3355 10.228099  779.7180 10.259448     0   0.469
4  7.1780729  7.23522456 24.268130 24.425546 40.97761 493.7062 40.95238 493.4022  736.2126  8.870031  735.7593  8.864570     0   0.352
5  1.9529454  1.99934371  8.672146  8.863748 62.89622 827.5818 62.86646 827.1902  755.0047  9.934272  754.6474  9.929571     0   1.380
6  0.1667439  0.08823883  2.152712  1.149959 31.34109 377.6035 31.36574 377.9004  593.3945  7.149331  593.8611  7.154953     0 179.000
7  0.5091390  0.47100278  2.948976  2.733108 36.95635 445.2572 36.97052 445.4279  779.6922  9.393882  779.9911  9.397483     0  67.400
8  0.2683852  0.19312645  2.791412  2.023014 22.72273 249.7004 22.73988 249.8888  485.0334  5.330038  485.3995  5.334060     0 426.000
    CONC3    CONC4    CONC5   CONC6   CONC7  CONC8   CONC9 CONC10 CONC11 CONC12 CONC13 CONCTIME1 CONCTIME2 CONCTIME3 CONCTIME4 CONCTIME5
1   1.430    1.250    1.660   2.730   3.580   3.60   2.510  2.220  1.720 0.5760 0.2050         0       0.5         1       1.5         2
2   0.350    0.352    0.393   0.407   0.563   1.30   0.824  0.532  0.384 0.1120 0.0677         0       0.5         1       1.5         2
3   0.524    0.419    0.405   0.397   0.895   1.08   0.541  0.347  0.191 0.0000 0.0000         0       0.5         1       1.5         2
4   0.385    0.301    0.367   0.543   0.862   1.46   0.847  0.569  0.387 0.0975 0.0000         0       0.5         1       1.5         2
5   1.480    1.410    1.620   2.030   3.190   2.92   1.640  1.020  0.655 0.0776 0.0000         0       0.5         1       1.5         2
6 399.000  383.000  254.000 135.000  69.000  39.20  24.400  4.680  1.080 0.3980 0.2810         0       0.5         1       2.0         4
7  80.000  150.000  126.000  73.900  57.000  42.00  41.500 19.000  5.910 1.6700 0.6530         0       0.5         1       2.0         4
8 758.000 1340.000 1050.000 605.000 310.000 178.00 130.000 27.600  9.010 3.0200 1.6600         0       0.5         1       2.0         4
  CONCTIME6 CONCTIME7 CONCTIME8 CONCTIME9 CONCTIME10 CONCTIME11 CONCTIME12 CONCTIME13 TIMEU AMOUNTU DOSEU VOLUMEU CONCU KELU  CLU     AUCU
1       2.5         4         8        12         16         24         48         72    HR      NA    MG       L NG/ML 1/HR L/HR NG.HR/ML
2       2.5         4         8        12         16         24         48         72    HR      NA    MG       L NG/ML 1/HR L/HR NG.HR/ML
3       2.5         4         8        12         16         24         48         72    HR      NA    MG       L NG/ML 1/HR L/HR NG.HR/ML
4       2.5         4         8        12         16         24         48         72    HR      NA    MG       L NG/ML 1/HR L/HR NG.HR/ML
5       2.5         4         8        12         16         24         48         72    HR      NA    MG       L NG/ML 1/HR L/HR NG.HR/ML
6       8.0        12        16        24         48         72         96        120    HR      NA    MG       L NG/ML 1/HR L/HR NG.HR/ML
7       8.0        12        16        24         48         72         96        120    HR      NA    MG       L NG/ML 1/HR L/HR NG.HR/ML
8       8.0        12        16        24         48         72         96        120    HR      NA    MG       L NG/ML 1/HR L/HR NG.HR/ML
        AUMCU    AUCNORMU AURCU CONCNORMU RATEU VOLUMEWU     CLWU    STUDY SITEID   SUBJID RAND TREATXT TRTCD PKCOLL PKBDFLD      PKTERM
1 NG.HR.HR/ML NG.HR/ML/MG    NA  NG/ML/MG    NA     L/KG ML/HR/KG B1261002   1001 10011002    2    3 mg     B  POINT  PLASMA PF-04634817
2 NG.HR.HR/ML NG.HR/ML/MG    NA  NG/ML/MG    NA     L/KG ML/HR/KG B1261002   1001 10011006    6    1 mg     A  POINT  PLASMA PF-04634817
3 NG.HR.HR/ML NG.HR/ML/MG    NA  NG/ML/MG    NA     L/KG ML/HR/KG B1261002   1001 10011003    3    1 mg     A  POINT  PLASMA PF-04634817
4 NG.HR.HR/ML NG.HR/ML/MG    NA  NG/ML/MG    NA     L/KG ML/HR/KG B1261002   1001 10011008    8    1 mg     A  POINT  PLASMA PF-04634817
5 NG.HR.HR/ML NG.HR/ML/MG    NA  NG/ML/MG    NA     L/KG ML/HR/KG B1261002   1001 10011003    3    3 mg     B  POINT  PLASMA PF-04634817
6 NG.HR.HR/ML NG.HR/ML/MG    NA  NG/ML/MG    NA     L/KG ML/HR/KG B1261002   1001 10011011   11  100 mg     E  POINT  PLASMA PF-04634817
7 NG.HR.HR/ML NG.HR/ML/MG    NA  NG/ML/MG    NA     L/KG ML/HR/KG B1261002   1001 10011020   20  900 mg     H  POINT  PLASMA PF-04634817
8 NG.HR.HR/ML NG.HR/ML/MG    NA  NG/ML/MG    NA     L/KG ML/HR/KG B1261002   1001 10011021   21  600 mg     G  POINT  PLASMA PF-04634817
  PERIODU PERIOD VISITU VISIT                            ACTTRT ACTTRTC             DOSEP DOSPND ROUTE RACES SEX  HT WT HTUNI WTUNI
1  PERIOD      2    DAY     1         PF-04634817 3 MG SOLUTION       B       PF-04634817   DONE  Oral BLACK   M 176 93    CM    KG
2  PERIOD      1    DAY     1         PF-04634817 1 MG SOLUTION       A       PF-04634817   DONE  Oral BLACK   M 183 99    CM    KG
3  PERIOD      1    DAY     1         PF-04634817 1 MG SOLUTION       A       PF-04634817   DONE  Oral ASIAN   M 175 76    CM    KG
4  PERIOD      1    DAY     1         PF-04634817 1 MG SOLUTION       A       PF-04634817   DONE  Oral BLACK   M 181 83    CM    KG
5  PERIOD      2    DAY     1         PF-04634817 3 MG SOLUTION       B       PF-04634817   DONE  Oral ASIAN   M 175 76    CM    KG
6  PERIOD      2    DAY     1       PF-04634817 100 MG SOLUTION       E       PF-04634817   DONE  Oral BLACK   M 174 83    CM    KG
7  PERIOD      2    DAY     1 PF-04634817 (FED) 100 MG SOLUTION       H PF-04634817 (FED)   DONE  Oral WHITE   M 187 83    CM    KG
8  PERIOD      1    DAY     1       PF-04634817 300 MG SOLUTION       G       PF-04634817   DONE  Oral WHITE   M 177 91    CM    KG
                                              PKRCOM
1 PKRCOM for Subject: 10011002 PKDATAROWID: 22289744
2 PKRCOM for Subject: 10011006 PKDATAROWID: 22289562
3 PKRCOM for Subject: 10011003 PKDATAROWID: 22289523
4 PKRCOM for Subject: 10011008 PKDATAROWID: 22289588
5 PKRCOM for Subject: 10011003 PKDATAROWID: 22289757
6 PKRCOM for Subject: 10011011 PKDATAROWID: 22290446
7 PKRCOM for Subject: 10011020 PKDATAROWID: 22291148
8 PKRCOM for Subject: 10011021 PKDATAROWID: 22290927
 [ reached 'max' / getOption("max.print") -- omitted 40 rows ]
```

**Comment by rudraya** [comment]
This issue was addressed in 22d43ea, please let me know if it still occurs.

**Comment by tensfeldt** [comment]
Test cases tc008 M1SS, M1SD, tc010 M1SD, tc011 M1SS, tc012 M1SS, tc013 M2SD, tc014 M2SS, tc015 M3SD, tc016 M3SS, tc018 M4SS ran successfully with commit 22d43ea and produced partial AUCs and incremental AUCs and generated the estimated concentration dataset as/where expected.

Testcase tc017 M4SD fails to execute but a separate issue #70 has been opened for this particular case. This issue #65 is being closed.
